tests/mir-opt/const_prop/slice_len.rs

```diff
@@ -8,7 +8,7 @@ fn main() {
     // CHECK-LABEL: fn main(
     // CHECK: debug a => [[a:_.*]];
     // CHECK: [[slice:_.*]] = copy {{.*}} as &[u32] (PointerCoercion(Unsize, AsCast));
-    // CHECK: assert(const true,
-    // CHECK: [[a]] = const 2_u32;
+    // CHECK-NOT: assert(const true,
+    // CHECK-NOT: [[a]] = const 2_u32;
```

Comment:
Seems odd to have tests asserting that an optimization does *not* happen. Shouldn't we instead keep the old assertions but comment them out?

Reply:
There is `COM:` for that,